src/lib/seam/components/CreateAccessCodeForm/CreateAccessCodeForm.tsx

```diff
@@ -18,6 +18,7 @@ import {
 
 export interface CreateAccessCodeFormProps extends CommonProps {
   deviceId: string
+  onSuccess?: () => void
```

Comment:
This will need to be
```suggestion
  onSuccess?: (accessCode: AccessCode) => void
```

Reply:
Sure, added in https://github.com/seamapi/react/pull/517/commits/1c3792bf0777bf0ad89778e19a9dded7fb1d4f14. Just a note that `EditAccessCodeFormProps`, however, won't get the `accessCode` back since it returns an action attempt.